Experiments_toolkit.py

```diff
@@ -57,151 +50,143 @@ def splitting_up_data(dictionary, dict_key, train, percent_train, test, percent_
     train_index = int(length * percent_train)
     test_index = int(length * percent_test)
-    train[dict_key] = dictionary[dict_key][0:train_index]
-    test[dict_key] = dictionary[dict_key][train_index:train_index + test_index]
-    validation[dict_key] = dictionary[dict_key][train_index + test_index:]
+    train[key] = data[0:train_index]
+    test[key] = data[train_index:train_index + test_index]
+    validation[key] = data[train_index + test_index:]
     return train, test, validation
 
 
-def preprocess_data(DATA, rows, filtering_params, window_function, window_size, offset):
+def preprocess_data(data, rows, filtering_params, window_function, window_size, offset):
     """
-       Description: Preprocess the data stored in DATA by windowing the traces and saving
+       Description: Preprocess the data stored in data by windowing the traces and saving
                                 these windows (numpy arrays) into a dictionary.
 
        Args:
-                DATA -- np array (Numpy array where the last row is time steps)
+            data -- list (List of numpy arrays where the last row is time steps, 
+            			  [(sensor_dims + 1), num_samples])
             rows -- int (Number of rows in the data)
             filtering_params -- All_Filtering_Params (Dataclass found in windowing.py that 
-                                                        stores important parameters for the 2 steps involved with 
-                                                        finding the precise start and end time of a lift in the data)
-            window_size -- int (Window size of the trace to be saved into the dictionary that will be
-                                                returned)
-            offset -- int (Number to add to the predicted start time of the lift from where the window
-                                                will start at)
+                                stores important parameters for the 2 steps involved with 
+                                finding the precise start and end time of a activity in the data)
+            window_size -- int (Window size of the trace to be saved into the
+            				    dictionary that will be returned)
+            offset -- int (Number to add to the predicted start time of the 
+            			   activity from where the window will start at)
 
        Return:
-           windowed_data_dictionary -- dict (Dictionary that stores the windows under the "windows" key)
+           windows-- int[] (Array of the windows cropped by the window_function)
            """
-    windowed_data_dictionary = {"windows": []}
-    for j in range(len(DATA)):
-        sample = DATA[j]
-        lift_windows = wl.initial_find_lift(
-            sample, rows, filtering_params.initial_filtering.window_size, filtering_params.initial_filtering.stride, filtering_params.initial_filtering.divisor)
-        if len(lift_windows) > 0:
+    windows = []
+    for j in range(len(data)):
```

Comment:
looks like you only use j to access the sample? if so, you could simplify by using

`for sample in data:`